Update the diagrams (#10077)
camptocamp/c2cgeoportal: doc/integrator/docker.rst

For OpenShift projects:

.. mermaid::

    graph LR

        client(Web client);

        k8s[Kubernetes routing];
        geoportal[GeoMapFish server<br>camptocamp/geomapfishapp-geoportal];
        tcc[TileCloud-chain<br>camptocamp/tilecloud-chain];
        tccs[TileCloud-chain slave<br>camptocamp/tilecloud-chain];
        mapserver[MapServer<br>camptocamp/mapserver<br>or/and<br>camptocamp/qgis-server];
        tinyows[tinyows];
        print[Mapfist Print<br>camptocamp/mapfish-print];

>         alembic[GeoMapFish alembic<br>camptocamp/geomapfishapp-geoportal<br>minor databage upgrade];

Reviewer: ```suggestion
        alembic[GeoMapFish alembic<br>camptocamp/geomapfishapp-geoportal<br>minor database upgrades];
```

>         config[Project config<br>camtocamp/%project%-config<br>injected the config in all the containers];

Reviewer: ```suggestion
        config[Project config<br>camtocamp/%project%-config<br>provides the config for all other containers];
```

        geoportal --> tinyows;
        geoportal ==> mapserver;
        geoportal -. cache, broadcast .-> redis;
        geoportal -.-> postgres;
        geoportal -. raster data .-> storage;
        tcc -- on the fly --> mapserver;
        tccs -- pregenertation --> mapserver;
        tcc -. queue .-> redis;
        tccs -. queue .-> redis;
        tcc -. tiles .-> storage;
        tccs -. tiles .-> storage;
        geoportal --> print;
        print --> geoportal;
        alembic -.-> postgres;
        mapserver -.-> postgres;
        mapserver -.-> storage;
        tinyows -.-> postgres;

        classDef data fill:#5ba6ff,color:black;
For standalone projects:

.. mermaid::

    graph LR

        client(Web client);

        apache[Apache];
        subgraph Project composition
        haproxy[haproxy];
        geoportal[GeoMapFish server<br>camptocamp/geomapfishapp-geoportal];
        tcc[TileCloud-chain<br>camptocamp/tilecloud-chain];
        tccs[TileCloud-chain slave<br>camptocamp/tilecloud-chain];
        mapserver[MapServer<br>camptocamp/mapserver<br>or/and<br>camptocamp/qgis-server];
        tinyows[tinyows];
        print[Mapfist Print<br>camptocamp/mapfish-print];

>         alembic[GeoMapFish alembic<br>camptocamp/geomapfishapp-geoportal<br>minor databage upgrade];

Reviewer: ```suggestion
        alembic[GeoMapFish alembic<br>camptocamp/geomapfishapp-geoportal<br>minor database upgrades];
```

>         config[Project config<br>camtocamp/%project%-config<br>injected the config in all the containers];

Reviewer: ```suggestion
        config[Project config<br>camtocamp/%project%-config<br>provides the config for all other containers];
```

Author: fixed :-)
